kernel/arch/dreamcast/kernel/panic.c

```diff
@@ -12,6 +13,6 @@
    can recover, call this. This is a pretty standard tactic from *nixy
    kernels which ought to be avoided if at all possible. */
 void arch_panic(const char *msg) {
-    printf("kernel panic: %s\r\n", msg);
+    fprintf(stderr, "kernel panic: %s\r\n", msg);
```

Comment:
We don't use stderr anywhere else in the kernel (except in malloc where it wasn't rewritten from the desktop version). Pretty sure if this should be anything it would be a dbglog, but it would be with a level to ensure it always shows up.